content/shmem_reductions.tex

```diff
@@ -376,6 +376,19 @@ \subsubsubsection{PROD}
   Zero on successful local completion. Nonzero otherwise.
 }
 
+\apinotes{
+    The binary operations performed by \openshmem reductions are intended to be
+    associative and commutative.
+    However, floating point arithmetic is not associative or commutative due to
+    the inherent inaccuracies of floating-point representations caused by
+    rounding errors, finite precision, and the order of the applied binary
+    operations.
```

Comment:
```suggestion
    rounding errors and finite precision.
```
The last part seems redundant since the sentence already says not associative or commutative.